CONTRIBUTING.md

```diff
@@ -255,9 +255,9 @@ or using AI-based tools:
    one for the project. For example, formatting with the standard tools is
    reasonable to assume without further comment. But if a PR is largely derived
    from running a specific Python script, regular expression, or AI-based tool
-   over the codebase, we ask that its description is transparent about this.
-   Ideally, this should include a prose description of how the tool was used to
-   formulate the change. Where relevant, we also encourage following the pattern
+   over the codebase, we ask that its description is transparent about this and
```

Comment:
Optional: replace the first "description" with something like "commit message" to avoid repeating "description" twice in this sentence.